File: src/com/android/orion/utility/StopWatch.java
package com.android.orion.utility;

public class StopWatch {
	long mStart = 0;
	long mStop = 0;
	double mInterval = 0;

	public StopWatch() {
	}

	public void start() {
		mStart = System.currentTimeMillis();
	}

	public void stop() {
		mStop = System.currentTimeMillis();
	}

	public double getInterval() {
		mInterval = (mStop - mStart) / 1000.0;
		return mInterval;
	}
}

File: app/src/main/java/com/android/orion/utility/StopWatch.kt
package com.android.orion.utility

import java.util.concurrent.TimeUnit

object StopWatch {
    private var mStart: Long = 0
    private var mStop: Long = 0
    private var mInterval = 0.0

    @JvmStatic
    fun start() {
        mStart = System.currentTimeMillis()
    }

    @JvmStatic
    fun stop() {
        mStop = System.currentTimeMillis()
        mInterval = (mStop - mStart) / (1.0 * TimeUnit.SECONDS.toMillis(1));
    }

    @JvmStatic
    fun getInterval(): Double {
        return mInterval
    }
}